config/config.go

```diff
@@ -25,8 +26,9 @@ func Get() (*Configuration, error) {
 	}
 
 	cfg = &Configuration{
-		BindAddr:  ":22000",
-		SecretKey: "FD0108EA-825D-411C-9B1D-41EF7727F465",
+		BindAddr:      ":22000",
+		DatasetAPIURL: "http://localhost:22000",
+		SecretKey:     "FD0108EA-825D-411C-9B1D-41EF7727F465",
 		MongoConfig: MongoConfig{
 			BindAddr:   "localhost:27017",
 			Collection: "datasets",
```

Comment:
Should collection be database?

Reply:
There is database and collection.....collection might not be wanted now there are several under one database.